frontend/coprs_frontend/alembic/script.py.mako

```diff
@@ -1,20 +1,24 @@
-"""
-${message}
+"""${message}
 
 Revision ID: ${up_revision}
-Revises: ${down_revision}
+Revises: ${down_revision | comma,n}
 Create Date: ${create_date}
-"""
 
-import sqlalchemy as sa
+"""
```

Comment:
The spacing here seems weird :-) can we continue having 

```
...
Create Date ...
"""

from alembic import op
```

Instead of

```
...
Create Date ...

"""
from alembic import op
```